.github/workflows/nightly-no-cache.yaml

```diff
@@ -50,6 +50,22 @@ jobs:
       - name: Test
         run: ./gradlew test -PtestJavaVersion=${{ matrix.java }} --stacktrace -x :smoke-tests:test -Porg.gradle.java.installations.paths=${{ steps.setup-test-java.outputs.path }} -Porg.gradle.java.installations.auto-download=false --no-build-cache
 
+  testLatestDep:
```

Comment:
I did not include this job into this workflow on purpose. Assuming that latestDep test does not use cache anyway.

Reply:
> Assuming that latestDep test does not use cache anyway.

I'll test that out 👍

Reply:
It looks like `testLatestDeps` does use the cache?

```
$ ./gradlew :instrumentation:spring:spring-webflux-5.0:javaagent:test -PtestLatestDeps=true

...

BUILD SUCCESSFUL in 1m 19s
68 actionable tasks: 5 executed, 63 up-to-date
S3 cache 300ms wasted on misses, reads: 3, elapsed: 300ms

$ ./gradlew :instrumentation:spring:spring-webflux-5.0:javaagent:test -PtestLatestDeps=true

...

BUILD SUCCESSFUL in 6s
68 actionable tasks: 68 up-to-date
```

Reply:
Interesting... I wonder if Gradle invalidates cache if new version is detected... But anyway it does make sense to include it in this workflow 👍